### PR TITLE
gui/main_menubar: Add the Open Textures Path menuitem.

### PR DESCRIPTION
**Pull request by Croden1999**
- gui/main_menubar: Add the Open Texture Folder menuitem for users edit textures.
- gui/app_context_menu: Add the Open/Delete Textures Folder menuitem.

Please devs check this PR.

### CODE REVIEW

**Comment by Zangetsu38** [comment]
suggestion, adding code for open folder of game in right click on it 👌

**Comment by Croden1999** [comment]
> suggestion, adding code for open folder of game in right click on it 👌

done

File: vita3k/gui/src/main_menubar.cpp [hunk 1]
```diff
@@ -26,10 +26,21 @@
...
         if (ImGui::MenuItem(lang["open_pref_path"].c_str()))
             open_path(emuenv.pref_path.string());
+        if (ImGui::MenuItem(lang["open_textures_path"].c_str())) {
+            if (!fs::exists(textures_path)) {
+                fs::create_directories(textures_path);
+                fs::create_directories(textures_path / "export");
+                fs::create_directories(textures_path / "import");
+                open_path(textures_path.string());
+            } else
+                open_path(textures_path.string());
```
Comment (EXtremeExploit) [inline vita3k/gui/src/main_menubar.cpp:42]: move this `open_path` and the one above outside the if exists check to get rid of duplicated code

**Review by EXtremeExploit — CHANGES_REQUESTED** [review]

File: vita3k/gui/src/main_menubar.cpp (hunk 1 above)
Comment (Croden1999) [reply vita3k/gui/src/main_menubar.cpp]: done

**Review by EXtremeExploit — APPROVED** [review]
lgtm

**Pull request merged**